src/utils/classifiers.ts

```diff
@@ -41,6 +41,11 @@ export const bitocracyUrl =
     ? 'https://bitocracy.sovryn.app'
     : 'https://bitocracy.test.sovryn.app';
 
+export const zeroUrl =
+  currentNetwork === AppMode.MAINNET
+    ? 'https://live.sovryn.app/zero'
+    : 'https://staging.sovryn.app/zero';
```

Comment:
According to the https://sovryn.monday.com/boards/2218344956/pulses/2707571652?doc_id=2707574694 , the URL for test env should be `https://test.sovryn.app/zero`.